### ISSUE TITLE
Ratatouille missing highlighting of interactive objects

### ISSUE THREAD

**Issue by TigiHof** [RPCS3/rpcs3#12174]
In Ratatouille interactive objects are highlighted with a blue border, this effect is missing in RPCS3. This problem was already reported in the forum in 2017 (https://forums.rpcs3.net/thread-181140.html), but obviously nobody ever opened a Github issue. I tried various graphics settings, but the problem remains.

This video capture shows how it is supposed to look like:
![Ratatouille_OK](https://user-images.githubusercontent.com/31275840/172057250-4b376fc2-e08a-486b-a2f7-b4b8229cead9.jpg)

This is how it actually looks in RPCS3:
![Ratatouille_Error](https://user-images.githubusercontent.com/31275840/172057268-ccea9c5b-4a05-4212-a2d2-03c0521e12a0.jpg)

Log:
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/8840206/RPCS3.log.gz)

RSX Capture:
[BLES00080_20220605165751_capture.zip](https://github.com/RPCS3/rpcs3/files/8840209/BLES00080_20220605165751_capture.zip)

**Comment by TigiHof** [comment]
Still an issue.

**Comment by kd-11** [comment]
There's a very high chance this is fixed with some aggressive accuracy options like WCB+WDB+RDB+RCB

**Comment by TigiHof** [comment]
> There's a very high chance this is fixed with some aggressive accuracy options like WCB+WDB+RDB+RCB

No, I had already tested these settings before opening the issue. I just tested the current release and the behaviour is unchanged. Neither WCB+WDB+RDB+RCB nor Accurate XFloat or Strict Rendering Mode are able to fix the problem.

**Comment by digant73** [comment]
@TigiHof is that still an issue?

**Comment by TigiHof** [comment]
> [@TigiHof](https://github.com/TigiHof) is that still an issue?

Yes, it is (tested with 0.0.38-18263).
